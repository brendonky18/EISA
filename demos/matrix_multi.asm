ADD R0, R0, 0x32
ADD R1, R1, 0x32
ADD R2, R2, 0x0
ADD R3, R3, 0x0
ADD R4, R4, 0x32
ADD R5, R5, 0x0
ADD R6, R6, 0x0
ADD R7, R7, 0x9f6
ADD R8, R8, 0x13ba
ADD R13, R13, 0x0
LDR R11, R4, R28
LDR R12, R7, R28
MULT R14, R11, R12
ADD R13, R13, R14
ADD R4, R4, 0x1
ADD R7, R7, R1
ADD R3, R3, 0x1
ADD R5, R5, 0x1
CMP R3, R1
BLT #0xc
ADD R2, R2, 0x1
ADD R6, R6, 0x1
SUB R3, R3, R3
SUB R5, R5, R5
STR R13, R8, R28
ADD R8, R8, 0x1
SUB R13, R13, R13
CMP R2, R0
BLT #0xc
END