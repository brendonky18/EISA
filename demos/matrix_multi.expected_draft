000001 00000 00000 1 000000000110010

000001 00001 00001 1 000000000110010

000001 00010 00010 1 000000000000000

000001 00011 00011 1 000000000000000

000001 00100 00100 1 000000000110010

000001 00101 00101 1 000000000000000

000001 00110 00110 1 000000000000000

000001 00111 00111 1 000100111110110

000001 01000 01000 1 001001110111010

000001 01101 01101 1 000000000000000

001101 01011 00100 0 11100 0000000000

001101 01100 00111 0 11100 0000000000

000100 01110 01011 0 01100 0000000000

000001 01101 01101 0 01110 0000000000

000001 00100 00100 1 000000000000001

000001 00111 00111 0 00001 0000000000

000001 00011 00011 1 000000000000001

000001 00101 00101 1 000000000000001

000011 00011 00001 0000000000000000

011110 1101 000000 1 000000000001100

000001 00010 00010 1 000000000000001

000001 00110 00110 1 000000000000001

000010 00011 00011 0 00011 0000000000

000010 00101 00101 0 00101 0000000000

001110 01101 01000 0 11100 0000000000

000001 01000 01000 1 000000000000001

000010 01101 01101 0 01101 0000000000

000011 00010 00000 0000000000000000

011110 1101 000000 1 000000000001100

10000000000000000000000000000000