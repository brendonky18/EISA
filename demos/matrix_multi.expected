00000100000000001000000000110010
00000100001000011000000000110010
00000100010000101000000000000000
00000100011000111000000000000000
00000100100001001000000000110010
00000100101001011000000000000000
00000100110001101000000000000000
00000100111001111000100111110110
00000101000010001001001110111010
00000101101011011000000000000000
00110101011001000111000000000000
00110101100001110111000000000000
00010001110010110011000000000000
00000101101011010011100000000000
00000100100001001000000000000001
00000100111001110000010000000000
00000100011000111000000000000001
00000100101001011000000000000001
00001100011000010000000000000000
01111011010000001000000000001100
00000100010000101000000000000001
00000100110001101000000000000001
00001000011000110000110000000000
00001000101001010001010000000000
00111001101010000111000000000000
00000101000010001000000000000001
00001001101011010011010000000000
00001100010000000000000000000000
01111011010000001000000000001100
10000000000000000000000000000000